### DISCUSSION TITLE
Add a not iplist capability.

### DISCUSSION THREAD

**Discussion by hpeters** [FoobarOy/foomuuri#34] [Ideas]
Some thing like:
saddr -@geoip drop

I can do it with the below now but it is a little less clean.
nft "ip saddr != @geoip_4 drop"
nft "ip6 saddr != @geoip_6 drop"

**Comment by kimheino** [comment]
Commit e457fa6. That was easy.

I'm using this myself:

```
   ssh saddr @geoip   # accept
   ssh drop           # drop everything else
```

It's identical to:

```
  ssh saddr -@geoip drop
  ssh
```

**Comment by hpeters** [comment]
Thanks again.
